dvc/api.py

```diff
@@ -96,7 +95,7 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
 
 @contextmanager
 def _make_repo(repo_url, rev=None):
-    if not repo_url or urlparse(repo_url).scheme == "":
+    if not repo_url or os.path.exists(repo_url):
```

Comment:
I'm using a solution suggested by @efiop , which identifies local repositories by checking if they are in the file system. There's no way a URL (e.g. `https://`, `git@github`) could be missed like that.

I'm not aware of any other edge case.

Reply:
Probably, `file://` but no one uses it :sweat_smile:

Reply:
Yeah, we don't support `file://` anywhere else, so no need to worry about it for now. Neither any user asked for it ever 🙂